src/main/java/com/theelixrlabs/healthcare/validation/Validator.java

```diff
@@ -62,6 +66,16 @@ public void validatePatientDto(PatientDto patientDto) throws CustomException {
         }
     }
 
+    /**
+     * Validates the patient name to ensure it is not blank.
+     * @param patientName The patient name to be validated.
+     */
+    public void validatePatientName(String patientName) {
+        if (StringUtils.isBlank(patientName)) {
+            throw new CustomException(messageUtil.getMessage(MessageConstants.PATIENT_NAME_CANNOT_BE_EMPTY));
```

Comment:
Instead of writing error message directly here, make this a generic validator, you can use the same method for string validation or patientName validation with a different error message.  
```suggestion
           public void validatePatientName(String patientName, String errorMessage) {
        if (StringUtils.isBlank(patientName)) {
            throw new CustomException(errorMessage);
            }
            }
```

Reply:
yes changed into a generic method to check for the String is not blank